<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerMappingsClass</class>
 <widget class="QDialog" name="CustomerMappingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NA22 Customer Mappings</string>
  </property>
  <widget class="QGroupBox" name="grpSearchCtriteria">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>4</y>
     <width>931</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Search Criteria</string>
   </property>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>17</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCustNo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customer-No:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCustNo">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpTransactionData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>941</width>
     <height>457</height>
    </rect>
   </property>
   <property name="title">
    <string>Customer Mappings</string>
   </property>
   <widget class="BasarTableWidget" name="twCustomerMappings">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>22</y>
      <width>911</width>
      <height>421</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="rowCount">
     <number>13</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>864</x>
     <y>530</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNew">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>New ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>530</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BasarTableWidget</class>
   <extends>QTableWidget</extends>
   <header>libbasarqtwidget_tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtCustNo</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>twCustomerMappings</tabstop>
  <tabstop>btnNew</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
